transformer_engine/pytorch/module/linear.py

```diff
@@ -108,7 +108,10 @@ def forward(
 
         if fp8:
             fp8_dtype_forward = get_fp8_te_dtype(fp8_meta["recipe"], fprop_tensor=True)
-            if not is_input_fp8:
+            if isinstance(inputmat, Float8Tensor):
+                inputmat_scale_inv = inputmat._scale_inv
+            else:
+                inputmat_scale_inv = torch.empty([1], dtype=torch.float32, device=inputmat.device)
```

Comment:
I think this is a merge conflict error. Should it be:
```suggestion
            if not is_input_fp8:
                if isinstance(inputmat, Float8Tensor):
                    inputmat_scale_inv = inputmat._scale_inv
                else:
                    inputmat_scale_inv = torch.empty([1], dtype=torch.float32, device=inputmat.device)
```

Reply:
Hi Tim, we have
```python
is_input_fp8 = isinstance(inp, Float8Tensor)
inputmat = inp.view(-1, in_features)
```
How could `is_input_fp8 == False` while `isinstance(inputmat, Float8Tensor) == True` in your suggestion?

Reply:
Ah, my mistake. The logic here makes sense to me now.